src/Blocks/BlockHandler.cpp

```diff
@@ -473,12 +473,14 @@ void cBlockHandler::DropBlock(cChunkInterface & a_ChunkInterface, cWorldInterfac
 				case E_BLOCK_COCOA_POD:
 				case E_BLOCK_CROPS:
 				case E_BLOCK_DARK_OAK_DOOR:
+				case E_BLOCK_DEAD_BUSH:
 				case E_BLOCK_DOUBLE_RED_SANDSTONE_SLAB:
 				case E_BLOCK_DOUBLE_STONE_SLAB:
 				case E_BLOCK_DOUBLE_WOODEN_SLAB:
 				case E_BLOCK_FIRE:
 				case E_BLOCK_FARMLAND:
 				case E_BLOCK_FLOWER_POT:
+				case E_BLOCK_GRASS:
```

Comment:
I believe this is incorrect, this is the grass block (dirt block with grass texture on the top side). Did you mean `E_BLOCK_TALL_GRASS`?